synapse/lib/storm.py

```diff
@@ -1581,31 +1805,22 @@ class MaxCmd(Cmd):
 
     def getArgParser(self):
         pars = Cmd.getArgParser(self)
-        pars.add_argument('name')
+        pars.add_argument('valu')
         return pars
 
     async def execStormCmd(self, runt, genr):
 
         maxvalu = None
         maxitem = None
 
-        func = self.getStormEval(runt, self.opts.name)
-
-        async for node, path in genr:
-
-            valu = func(path)
-            if valu is None:
-                continue
+        async for item in genr:
 
-            # Specifically if the name given is a ival property,
-            # we want to max on upper bound of the ival.
-            prop = node.form.prop(self.opts.name)
-            if prop and isinstance(prop.type, s_types.Ival):
-                valu = valu[1]
+            valu = await s_stormtypes.toprim(self.opts.valu)
+            valu = s_stormtypes.intify(valu)
```

Comment:
does this properly handle intervals? (this is really a marker for myself to double check things)

Reply:
No, and i dont think it should.  ultimately, it would be so easy to be specific rather than having ambiguous definition of what it does.

Reply:
Probably need to add something about it to the storm command usage change log or revert to old behavior